src/org/ods/component/RolloutOpenShiftDeploymentOptions.groovy

```diff
@@ -47,6 +47,13 @@ class RolloutOpenShiftDeploymentOptions extends Options {
      * directory referenced by `chartDir` exists. */
     List<String> helmValuesFiles
 
+    /**
+     * List of paths to values files (empty by default). Only relevant if the
+     * directory referenced by `chartDir` exists.
+     * These must contain a suffix called '.env.yml' - which will be replaced
```

Comment:
Is this meant to say that `env` gets replaced? If so, maybe change the notation to e.g. `.<env>.yml' - that would make it clearer for me.

Reply:
Forget my comment, understood how it works now. Maybe we can make an example here that tells users they need to create values.dev.yaml, values.test.yaml and values.prod.yaml - and then if they pass values.env.yaml here, those files will be picked up.